v3.1.6 it.json

GregoryWarn opened the pull request:
Updated Italian Localization

Haxxer commented:
Thank you for the contribution - most people utilize weblate, as it provides an easy way to collaborate with others when localizing modules:
https://weblate.foundryvtt-hub.com/projects/item-piles/

GregoryWarn commented:
Oh, I know, I am the person that brought italian localization from 0 to 100% in Weblate - ItemPiles...
In the meanwhile, during my sessions I discovered some typos and fixed them on my system... Unfortunately I don't remember exactly what I fixed, so I copied my PC's it.json file to github...
